### PR TITLE
Improve gradle user friendliness

### LINKED ISSUE #173: CI : create a common gradle task to build all Artefact
Need : commiters and CI jobs should share easy commands to build all artifacts

This gradle task named 'installKeyple' will prioritize the build of keyple-core task before the other artifact
Each module will define the behaviour attached to this task.
Executing this task on a master gradle project will result on the execution of this task on all submodule

### LINKED ISSUE #201: CI : remove .circleci folder
CI configuration is now hosted in eclipse/keyple-ops repository

- [ ] remove : /.circleci /JenkisFile
- [ ] add link to keyple-ops repository Readme.md

### LINKED ISSUE #204: CI : commit gradle wrapper to repo
for keyple-java/ ** /**
each project in keyple-java/example/** /**

### PR DESCRIPTION
**Pull request by odelcroi**
fixes #204
fixes #173 
fixes #201

### CODE REVIEW

File: java/component/build.gradle [hunk 1]
```diff
@@ -1,42 +1,39 @@
...
-
-    aggregatedJavaProjects.each { aggregatedJavaProject ->
-        source += aggregatedJavaProject.javadoc.source
-        classpath += aggregatedJavaProject.javadoc.classpath
-        excludes += aggregatedJavaProject.javadoc.excludes
-        includes += aggregatedJavaProject.javadoc.includes
-
-    }
-}
+////create task to agregate javadoc
```
Comment (brice-ruppen) [inline java/component/build.gradle]: Commented legacy should not be commited.

File: README.md [hunk 2]
```diff
@@ -130,18 +130,14 @@ Here are the prerequisites to build the keyple components (jars)
 #### Linux or Macos
 Following commands will build all the artifacts at once. The first command is required to be executed at least once to build the gradle wrapper. Then, the keyple-core artifact is built and installed into the local maven repository as other artifacts depends on it.
```
Comment (curious-jeremy) [inline README.md]: The part: _The first command is required to be executed at least once to build the gradle wrapper_ can be deleted, has the wrapper is now included

File: java/example/generic/pc/Demo_ObservableReaderNotification/build.gradle [hunk 3]
```diff
@@ -11,6 +13,19 @@ ext {
...
 }
 
+mainClassName = 'org.eclipse.keyple.example.generic.pc.Demo_ObservableReaderNotification.Demo_ObservableReaderNotification_Pcsc'
+
+task('runPcsc', dependsOn: 'classes', type: JavaExec, group:'application') {
+    main = 'org.eclipse.keyple.example.generic.pc.Demo_ObservableReaderNotification.Demo_ObservableReaderNotification_Pcsc'
+    classpath = sourceSets.main.runtimeClasspath
+}
+
+task('runStub', dependsOn: 'classes', type: JavaExec, group:'application') {
```
Comment (curious-jeremy) [inline java/example/generic/pc/Demo_ObservableReaderNotification/build.gradle]: Very handy this task, for people running the project in a terminal

File: android/README.md [hunk 4]
```diff
@@ -1,23 +1,43 @@
-## Building Keyple Android components
-
-If you want to build the keyple android components (aar plugins, apk example app), you need : 
+### Android components
+If you want to build the keyple android components (aar plugins), you need : 
 - Java JDK 1.8 or newer (OMAPI app requires java 1.8)
```
Comment (curious-jeremy) [inline android/README.md]: about the _or newer_ I don't know: what I know is that it won't work with java 11

**Pull request merged**